src/__snapshots__/mackerel-container-agent-definition.test.ts.snap

```diff
@@ -132,7 +115,6 @@ Object {
         ],
         "Family": "TaskDefinition",
         "NetworkMode": "bridge",
-        "PlacementConstraints": Array [],
```

Comment:
https://github.com/awslabs/aws-cdk/pull/1646

(I think this p-r title/ description isn't kind enough.  I never imagined it introduces output change to pure ECS stacks...)